backend/fms_core/viewsets/readset.py

```diff
@@ -0,0 +1,22 @@
+
+from rest_framework import viewsets
+from rest_framework.response import Response
+from rest_framework.decorators import action
+
+from fms_core.filters import ReadsetFilter
+from fms_core.models.readset import Readset
+from fms_core.serializers import ReadsetSerializer
+
+from ._utils import _list_keys
+from ._constants import _readset_filterset_fields
+
+class ReadsetViewSet(viewsets.ModelViewSet):
+    queryset = Readset.objects.all()
+
+    serializer_class = ReadsetSerializer
+
+    ordering_fields = (
+        *_list_keys(_readset_filterset_fields),
+    )
+
+    filterset_class = ReadsetFilter
```

Comment:
define filterset_fields here instead of the the filterset_class unless you need specific filters.